src/calendar.cpp

```diff
@@ -398,6 +398,14 @@ int calendar::season_length()
     return cached_season_length;
 }
 
+int calendar::turn_of_year() const {
+	return (season_turns() * season) + (turn_number % season_turns());
```

Comment:
You're right, I must have missed something.

You should replace the tabs with 4 spaces (see doc/CODE_STYLE.txt) and apply the style to changed code (you can use `astyle` for this).